cellprofiler/modules/measureimageskeleton.py

```diff
@@ -5,12 +5,19 @@
 ====================
 
 **MeasureImageSkeleton** measures the number of branches and endpoints in a
-morphological skeleton. A branch is a pixel with more than two neighbors
-and and an endpoint is a pixel with only one neighbor. You can create a
-morphological skeleton with the **MorphologicalSkeleton** module from the
-*Advanced* category. You can use the ouput of this module for graph theoretic 
-analysis; you may also use it to analyze the number of total branches and 
-endpoints for branching objects (such as neurons) in an image.
+morphological skeleton. You can create a morphological skeleton with the
```

Comment:
Looks great! Can you please just add a new paragraph after the first sentence? We try to have one-liners to describe the module and the first sentence suits that goal perfectly.